src/sentry/static/sentry/app/components/panels/panelBody.jsx

```diff
@@ -18,7 +18,7 @@ const PanelBody = ({className, disablePadding, flex, direction, ...props}) => {
 
   return (
     <Comp
-      className={cx(padding, textClassName, className)}
+      className={cx(className, padding, textClassName)}
```

Comment:
turns out, emotion treats this differently with multiple emotion generated class names.

Reply:
but this will get refactored in emotion10 upgrade anyway